nemo/collections/asr/models/sortformer_diar_models.py

```diff
@@ -109,14 +149,19 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
         self._init_loss_weights()
 
         self.eps = 1e-3
+        self.negative_init_val = -99
         self.loss = instantiate(self._cfg.loss)
 
+        self.async_streaming = self._cfg.get("async_streaming", False)
         self.streaming_mode = self._cfg.get("streaming_mode", False)
         self.save_hyperparameters("cfg")
         self._init_eval_metrics()
         speaker_inds = list(range(self._cfg.max_num_of_spks))
         self.speaker_permutations = torch.tensor(list(itertools.permutations(speaker_inds)))  # Get all permutations
 
+        self.max_batch_dur = self._cfg.get("max_batch_dur", 20000)
```

Comment:
Could you share more details on how you chose the default value for max_batch_dur please? This would be GPU memory dependent right? Would 20000 be low enough to not cause OOM in line 510 on most of the supported GPUs?

Reply:
This needs to be revisited when we upload test and examples codes